### PR TITLE
Document the replications settings in app_Settings

### PR DESCRIPTION
**Pull request by newtewt**
Moved the medic-docs documentation about .replications for #284

### CODE REVIEW

File: content/en/apps/reference/app-settings/replication.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: ".replications"
+linkTitle: ".replications"
+weight: 5
+description: >
+  **Replications**: Configure replication of databases to a main meta database.
+keywords: replications
+---
+
+
+**Deprecation note: This is only available in versions 3.5.0 to 3.9.0. As of 3.10.0 this no longer configurable and happens nightly for user meta databases to a central meta data database.**
```
Comment (abbyad) [inline content/en/apps/reference/app-settings/replication.md:11]:
```suggestion
{{% alert title="Deprecated" %}} 
The `replications` field is only available in versions 3.5.0 to 3.9.0. As of 3.10.0 this field is ignored, and replication happens nightly for user meta databases to a central meta data database.
{{% /alert %}}
```

File: content/en/apps/reference/app-settings/replication.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: ".replications"
+linkTitle: ".replications"
+weight: 5
+description: >
+  **Replications**: Configure replication of databases to a main meta database.
```
Comment (abbyad) [inline content/en/apps/reference/app-settings/replication.md:6]:
```suggestion
  **Replications** [deprecated]: Configure replication of databases to a main meta database.
```

File: content/en/apps/reference/app-settings/replication.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: ".replications"
```
Comment (abbyad) [inline content/en/apps/reference/app-settings/replication.md:2]:
```suggestion
title: ".replications [deprecated]"
```

**Review by abbyad — CHANGES_REQUESTED** [review]
Looks good! I could have used this page last night :)

I added some suggestions to make it clearer that this field is deprecated.

Also, should the file name match the field name exactly?

**Comment by newtewt** [comment]
@abbyad  yeah I updated the name to match and thanks for the suggestions!

**Review by abbyad — APPROVED** [review]
Nice work!

**Pull request merged**